### ISSUE TITLE
Group Chat?

### ISSUE THREAD

**Issue by AxesandGrinds** [AgoraIO/RTM#47]
Hi,

I have a simple question. Can more than 2 users use the same channel in the rtm chat? In other words, can users use this as group chat?

Thanks

**Comment by AxesandGrinds** [comment]
I see the radio button in activity_selection.xml which allows to peer to peer or group/channel mode.

**Comment by plutoless** [comment]
yes you can. you can also use channel message to broadcast a message to all people in the channel
